osgar/drivers/usbcam.py

```diff
@@ -0,0 +1,47 @@
+"""
+  Publish jpg images captured using libuvc (pupil-labs fork).
+  Only cameras directly supporting jpg output are supported.
+"""
+
+import uvc
+from threading import Thread
+
+from osgar.bus import BusShutdownException
+
+
+class UsbCam:
+    def __init__(self, config, bus):
+        self.input_thread = Thread(target=self.run_input, daemon=True)
+        self.bus = bus
+
+        bus = config.get('bus', 1)
+        port = config.get('port', 0)
+        self.sleep = config.get('sleep', 10)
+        
+        dev_list = uvc.device_list()
+        for dev in dev_list:
+            if dev['bus_number'] == bus and dev['bus_ports'][0] == port:
+                self.cap = uvc.Capture(dev["uid"])
+                self.cap.frame_mode = self.cap.avaible_modes[0]
+                self.cap.bandwidth_factor = 1.2
```

Comment:
I am not sure what factor 1.2 means - can you add note what it is and what we should do with this number is something (what?) happens?

Reply:
I am not sure either so no, I cannot add note what it is. When this number is too high, you get "not enough space on device" and when it is too low you get only a partial picture (top part is usually ok, bottom part is usually grey).